backend/src/main/routes/ProjectRoute.ts

```diff
@@ -7,21 +7,26 @@ import HttpException from '../exceptions/HttpException';
 import { WorksonService } from '../services/WorksOnService';
 import { checkJwt, checkRole } from '../middleware/JWTMiddleware';
 import { Roles } from '../security/Roles';
+import { EmployeeAccountService } from '../services/EmployeeAccountService';
 
 @injectable()
 export default class ProjectRoute extends CommonRoutesConfig {
   constructor(
     @inject('express-app') app: express.Application,
     private projectService: ProjectService,
-    private worksOn: WorksonService
+    private worksOn: WorksonService,
+    private employeeAccountService: EmployeeAccountService
   ) {
     super(app, 'ProjectRoute');
   }
 
   configureRoutes(): express.Application {
     this.getApp()
       .route(`/project`)
-      .all(checkJwt, checkRole(new Set([Roles.SUPERVISOR, Roles.BUSINESS])))
+      .all(
+        checkJwt
+        // checkRole(new Set([Roles.SUPERVISOR, Roles.BUSINESS]))
```

Comment:
I think you add back the `checkRole` as it is still working. But we might have to re-think how we want to approach checking the role. We'd need to check if we can increase the rate limit from Auth0 or check our own solution.

Reply:
ok sure